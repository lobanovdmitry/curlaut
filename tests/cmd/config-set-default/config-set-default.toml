args = "config set-default local-non-default"
fs.cwd = "fs"

[env.add]
HOME = "."
