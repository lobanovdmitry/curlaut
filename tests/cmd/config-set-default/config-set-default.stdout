Set keycloak by default alias local-non-default
